src/filters/BlendImage.ts

```diff
@@ -39,20 +39,27 @@ export const blendImageDefaultValues: Partial<TClassProperties<BlendImage>> = {
  *
  * const filter = new BlendImage({
  *  image: fabricImageObject,
- *  mode: 'multiply',
- *  alpha: 0.5
+ *  mode: 'multiply'
  * });
  * object.filters.push(filter);
  * object.applyFilters();
  * canvas.renderAll();
  */
 export class BlendImage extends BaseFilter {
   /**
-   * Color to make the blend operation with. default to a reddish color since black or white
-   * gives always strong result.
+   * Image to make the blend operation with.
    **/
   declare image: FabricImage;
 
+  /**
+   * Blend mode for the filter: either 'multiply' or 'mask'. 'multiply' will
+   * multiply the values of each channel (R, G, B, and A) of the filter image by
+   * their corresponding values in the base image. 'mask' will only look at the
+   * alpha channel of the filter image, and apply those values to the base
+   * image's alpha channel.
```

Comment:
I m not 100% sure this is the best description, but if you change it is probably mirroring the reality at the moment.
I think we should link to mdn pages that explain what multiply or mask would do and then stick our implementation to those.